source/_components/light.everlights.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "EverLights Light"
+description: "Instructions on how to setup EverLights within Home Assistant."
+date: 2018-12-29
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+featured: false
+ha_release: 0.86
+---
+
+[EverLights](https://myeverlights.com/) are permanent Christmas lights installed on a home's gutters or flashing. This integration can change all LEDs in a zone to a single color or activate a pattern that was previously saved to the control box.
+
+### {% linkable_title Configuration Details %}
+
+To enable EverLights, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: everlights
+    hosts:
+    - 192.168.1.123
+    - 192.168.1.124
+    ...
+```
+
+### {% linkable_title Effects %}
```

Comment:
Add a configuration tag section here before this header.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration